app/models/parser/twitter_search_item.rb

```diff
@@ -0,0 +1,36 @@
+module Parser
+  class TwitterSearchItem < Base
+    include ProviderTwitter
+
+    TWITTER_SEARCH_ITEM_URL = /https:\/\/twitter\.com\/search[^\s"]*/
```

Comment:
Request: I suggest we make this regular expression broader:

- We should always expect at least the `q` parameter
- We should support HTTP
- We need to be sure it starts with http
- We should support other subdomains (www, mobile, etc.)

That said, I suggest to use the host part we have here: https://github.com/meedan/pender/blob/develop/app/models/parser/twitter_item.rb#L5

Suggestion: We could extract the host part of that regular expression into a `TWITTER_HOST_URL` constant defined in `ProviderTwitter` and reused by the other Twitter modules.

Reply:
Thank you, these were great suggestions. I have implemented everything except the last suggestion, which I will move later.